### PR TITLE
added an alternative way of installation via conda

### PR DESCRIPTION
**Pull request by prasanna28d**

### CODE REVIEW

**Comment by ekshaks** [comment]
- can you update your fork to latest commit in ragpipe/main
- at this stage, I want the requirements.txt to be very lean. Can you keep only the dependencies from `pyproject.toml`, without version numbers, and see if that is sufficient to install?

**Comment by prasanna28d** [comment]
1. Yes its updated to have the latest commit - llm_keys in it
2. Tried installing with just the packages in the pyproject.toml file, but still got import issues with - torch, transformers, jsonlines & diskcache.

File: README.md [hunk 1]
```diff
@@ -103,7 +117,6 @@ examples/insurance/
 python -m examples.insurance.insurance
 ```
 
-The default LLM is [Groq](https://groq.com/). Please set GROQ_API_KEY in `.env`. Alternatively, openai LLMs (set `OPENAI_API_KEY`) and ollama based local LLMs (`ollama/..` or `local/..`) are also supported.
```
Comment (ekshaks) [inline README.md]: why is this removed?

Comment (prasanna28d) [reply README.md]: oops! seemed to have messed up the merge. is fixed now

**Review by ekshaks — APPROVED** [review]
LGTM

**Comment by ekshaks** [comment]
ok.
